include/alpaca/detail/types/glm_vector.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#ifndef ALPACA_GLM_VECTOR_SUPPORT
```

Comment:
Shouldn't this be `ifdef`? 

Why is this `ifndef`?  

Please rename the macro `ALPACA_INCLUDE_SUPPORT_GLM_VECTOR` and make it opt-in, not opt-out.